### ISSUE TITLE
Can WDL runner be restarted on google cloud?

### ISSUE THREAD

**Issue by wenweiliang** [openwdl/wdl#218]
Hi, 
My question seem naive but I want to make sure about it before I dive into it.  Does the wdl runner restart from where it failed last time? It seems feasible because google cloud backend support the restart feature. Thanks!

**Comment by geoffjentry** [comment]
Hi @wenweiliang - that's not something WDL is opinionated about and it's not a WDL feature. However, most (all?) implementations support this behavior. What are you using to run your WDLs?

**Comment by wenweiliang** [comment]
I am using Cromwell

**Comment by wenweiliang** [comment]
Sorry let me rephrase my question. I would like to use wdl runner (https://github.com/openwdl/wdl/tree/master/runners/cromwell_on_google/wdl_runner) on google cloud to run my workflow like step 1 to step 10. Let's say my first attempt of workflow running stops at step 5, does wdl runner can resume work from step 5 at my second attempt? Thanks.

**Comment by geoffjentry** [comment]
Cromwell can do this but I don't believe that behavior is enabled in wdl_runner.

Also note that the presence of wdl_runner in this repo is a historical artifact, it's in the process of being moved. I'm trying to track down who the right person to contact would be for support
